src/trans.rs

```diff
@@ -574,20 +582,20 @@ impl<'v, 'tcx: 'v> BinaryenFnCtxt<'v, 'tcx> {
             // TODO: when it's possible to print characters or interact with the environment, also
             //       handle #[lang = "panic_fmt"] to support panic messages
             let mut is_fn_panic = false;
-            if self.sig.output == ty::FnOutput::FnDiverging {
-                let attrs = self.tcx.map.attrs(nid);
-                for attr in attrs {
-                    if let MetaItemKind::NameValue(ref node, ref span) = attr.node.value.node {
-                        if node == "lang" {
-                            if let LitKind::Str(ref value, _) = span.node {
-                                if value == "panic" {
-                                    is_fn_panic = true;
-                                }
-                            }
-                        }
-                    }
-                }
-            }
+            // if self.sig.output == ty::FnOutput::FnDiverging {
```

Comment:
It wasn't clear what to do in this case.  FnOutput was removed in a recent refactor (see: https://github.com/rust-lang/rust/commit/ed02344fbc7d3bed591f528304f603f9d320e919).